src/test/ui/methods/method-not-found-generic-arg-elision.stderr

```diff
@@ -0,0 +1,93 @@
+error[E0599]: no method named `distance` found for struct `Point<i32>` in the current scope
+  --> $DIR/method-not-found-generic-arg-elision.rs:82:23
+   |
+LL | struct Point<T> {
+   | --------------- method `distance` not found for this
```

Comment:
Looking at this with fresh eyes in context, it might make sense to expand this label to be `method distance not found for Point when T is i32, but it is found when T is f64`, but maybe that's too verbose.